packages/atlogger/src/atlogger.c

```diff
@@ -10,7 +10,7 @@
 #define PREFIX_BUFFER_LEN 64
 #define INFO_PREFIX "\e[0;32m[INFO]\e[0m"
 #define WARN_PREFIX "\e[0;31m[WARN]\e[0m"
-#define ERROR_PREFIX "\e[1;31m[ERRO]\e[0m"
+#define ERROR_PREFIX "\e[1;31m[ERROR]\e[0m"
```

Comment:
I think making `ERRO` longer will no longer align the logs nicely

Reply:
'ERRO' would align, but users would think that's a typo. We could use something like 'ERRR', what do you think?

Reply:
Could add a space after the other ones

Reply:
Anything works! Not a huge deal

Reply:
In that case, I'm leaving it as 'ERROR' so that it's clearer.